gettsim/data/soz_vers_beitr.yaml

```diff
@@ -1,1192 +1,749 @@
-    en:
-  description:
-    de:
+    de: https://de.wikipedia.org/wiki/Bezugsgr%C3%B6%C3%9Fe
     en:
-  values:
-    1984-01-01:
-      value: 0
-      note:
-    1999-01-01:
-      value: 0.1
-      note:
-    2004-01-01:
-      value: 0.11
-      note:
-    2007-01-01:
-      value: 0.13
-      note:
-mini_ag_grv:
+  1984-01-01:
+    west: 1396
+    ost: 0
+  1985-01-01:
+    deviation_from: previous
+    west: 1432
+  1986-01-01:
+    deviation_from: previous
+    west: 1467
+  1987-01-01:
+    deviation_from: previous
+    west: 1539
+  1988-01-01:
+    deviation_from: previous
+    west: 1575
+  1989-01-01:
+    deviation_from: previous
+    west: 1611
+  1990-01-01:
+    west: 1682
+    ost: 715.8086336747059
+  1991-01-01:
+    west: 1718
+    ost: 787.3894970421765
+  1992-01-01:
+    west: 1790
+    ost: 1073.712950512059
+  1993-01-01:
+    west: 1897
+    ost: 1395.826835665677
+  1994-01-01:
+    west: 2004
+    ost: 1574.778994084353
+  1995-01-01:
+    west: 2076
+    ost: 1682.150289135559
+  1996-01-01:
+    west: 2112
+    ost: 1789.521584186765
+  1997-01-01:
+    west: 2183
+    ost: 1861.102447554235
+  1998-01-01:
+    west: 2219
+    ost: 1896.892879237971
+  1999-01-01:
+    west: 2255
+    ost: 1896.892879237971
+  2000-01-01:
+    west: 2291
+    ost: 1861.102447554235
+  2001-01-01:
+    deviation_from: previous
+    ost: 1932.683310921706
+  2002-01-01:
+    west: 2345
+    ost: 1960
+  2003-01-01:
+    west: 2380
+    ost: 1995
+  2004-01-01:
+    west: 2415
+    ost: 2030
+  2006-01-01:
+    west: 2450
+    ost: 2065
+  2007-01-01:
+    deviation_from: previous
+    ost: 2100
+  2008-01-01:
+    deviation_from: previous
+    west: 2485
+  2009-01-01:
+    west: 2520
+    ost: 2135
+  2010-01-01:
+    west: 2555
+    ost: 2170
+  2011-01-01:
+    deviation_from: previous
+    ost: 2240
+  2012-01-01:
+    deviation_from: previous
+    west: 2625
+  2013-01-01:
+    west: 2695
+    ost: 2275
+  2014-01-01:
+    note: http://www.bundesregierung.de/ContentArchiv/DE/Archiv17/Artikel/2013/10/2013-10-16-rechengroessen-sozialversicherung.html
+    west: 2765
+    ost: 2345
+  2015-01-01:
+    west: 2835
+    ost: 2415
+  2016-01-01:
+    note: https://www.bundesregierung.de/Content/DE/Artikel/2015/10/2015-10-14-sozialversicherung.html
+    west: 2905
+    ost: 2520
+  2017-01-01:
+    note: https://www.bundesregierung.de/Content/DE/Artikel/2016/10/2016-10-12-bemessunggrenzen-sozialversicherung.html
+    west: 2975
+    ost: 2660
+  2018-01-01:
+    west: 3045
+    ost: 2695
+  2019-01-01:
+    note: Sozialversicherungs-Rechengrößenverordnung 2019, BGBl I S. 2024, 27.11.2018
+    west: 3115
+    ost: 2870
+  2020-01-01:
+    reference: §2 V. v. 17.12.2019 BGBl. I S. 2848
+    west: 3185
+    ost: 3010
+
+eink_grenzen:
   name:
-    de: Pauschalabgabe GRV bei geringf. Beschäftigung  § 168 I Nr. 1b SGB VI
+    de: Midijob-Grenze  (Höhe § 20 SGB IV (2)) und Minijob Grenze (bis 1999 unterschiedliche Grenzen)
     en:
```

Comment:
Thresholds for marginal employment ('mini' and 'midi' jobs)

Reply:
Added.